Azure/azure-cli
# [Network] `az network watcher connection-monitor create`: Add VMSS support in CM source/destination

**Related command**

az network watcher connection-monitor create

**Description**

Azure VMSS support is now available in connection-monitor.  Support for VMSS as connection-monitor endpoints is now availbale. This change will enable customer to create connection-monitor with VMSS as endpoints via Azure CLI.

New parameters are added -

--endpoint-source-address-include- Space-separated list of IP addresses to include in CM endpoint
--endpoint-source-address-exclude- Space-separated list of IP addresses to exclude in CM endpoint


**Testing Guide**
- Run the command to verify Connection monitor is getting created with VMSS as source/destination- 
az network watcher connection-monitor create -n cmVMSSCli --endpoint-source-name vmssSource --endpoint-source-type AzureVMSS --endpoint-source-resource-id /subscriptions/**************************/resourceGroups/rg-eastus2euap/providers/Microsoft.Compute/virtualMachineScaleSets/vmss-eastus2euap --endpoint-source-address-include 10.69.0.4 --endpoint-source-address-exclude 10.69.0.6 --endpoint-dest-name vmssdest --endpoint-dest-resource-id /subscriptions/**********************/resourceGroups/rg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss --endpoint-dest-address-include 10.128.0.4 --endpoint-dest-address-exclude 10.128.0.5 --endpoint-dest-type AzureVMSS --test-config-name tcptc --protocol Tcp --tcp-port 2048

**History Notes**


[Component Name 1] BREAKING CHANGE: `az command a`: Make some customer-facing breaking change
[Component Name 2] `az command b`: Add some customer-facing feature

---

This checklist is used to make sure that common guidelines for a pull request are followed.
